sdk/search/Azure.Search.Documents/src/Options/SemanticSearchOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Search.Documents.Models
+{
+    /// <summary>
+    /// Options for performing Semantic Search.
+    /// </summary>
+    public partial class SemanticSearchOptions
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SemanticSearchOptions"/> class.
+        /// </summary>
+        public SemanticSearchOptions()
+        {
+            QueryAnswer = new QueryAnswer();
+            QueryCaption = new QueryCaption();
+        }
+
+        /// <summary> The name of a semantic configuration that will be used when processing documents for queries of type semantic. </summary>
+        public string SemanticConfigurationName { get; set; }
+
+        /// <summary>
+        /// This parameter is only valid if the query type is 'semantic'. If set, the query returns answers extracted from key passages in the highest
+        /// ranked documents.The number of answers returned can be configured by appending the pipe character '|' followed by the 'count-(number of answers)'
+        /// option after the answers parameter value, such as 'extractive|count-3'. Default count is 1. The confidence threshold can be configured by appending
+        /// the pipe character '|' followed by the 'threshold-(confidence threshold)' option after the answers parameter value, such as 'extractive|threshold-0.9'.
+        /// Default threshold is 0.7.
+        /// </summary>
+        public QueryAnswer QueryAnswer { get; set; }
+
+        /// <summary>
+        /// This parameter is only valid if the query type is 'semantic'. If set, the query returns captions extracted from key passages in the highest
+        /// ranked documents. When Captions is set to 'extractive', highlighting is enabled by default, and can be configured by appending the pipe
+        /// character '|' followed by the 'highlight-(true/false)' option, such as 'extractive|highlight-true'. Defaults to 'None'.
+        /// </summary>
+        public QueryCaption QueryCaption { get; set; }
+
+        /// <summary> Allows the user to choose whether a semantic call should fail completely (default / current behavior), or to return partial results. </summary>
+        public SemanticErrorMode? SemanticErrorMode { get; set; }
+
+        /// <summary> Allows the user to set an upper bound on the amount of time it takes for semantic enrichment to finish processing before the request fails. </summary>
+        public TimeSpan? SemanticMaxWaitInMilliseconds { get; set; }
```

Comment:
We can drop the `Semantic` prefix on this one and don't need the `InMilliseconds` if it's a TimeSpan.  I think `TimeSpan? MaxWait { get; set; }` would be sufficient.